hyundai_kia_connect_api/ApiImpl.py

```diff
@@ -57,10 +57,6 @@ def refresh_vehicles(self, token: Token, vehicles: list[Vehicle]) -> None:
         Required for Kia USA as key is session specific"""
         return vehicles
 
-    def get_last_updated_at(self, value) -> dt.datetime:
```

Comment:
Moved this into `utils` instead, as there was a duplicate implementation for every region